tests/h/search/core_test.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import datetime
+
+import pytest
+
+from h import search
+
+
+class TestWithoutSeparateReplies(object):
+    """Tests for what happens when no separate_replies argument is given."""
+
+    def test_it_returns_replies_in_annotations_ids(self, matchers, pyramid_request, Annotation):
+        """Without separate_replies it returns replies in annotation_ids.
+
+        Test that if no separate_replies argument is given then it returns the
+        ids of replies mixed in with the top-level annotations in
+        annotation_ids.
+        """
+        annotation = Annotation(shared=True)
```

Comment:
By default `factories.Annotation` creates private annotations. I considered making a `factories.SharedAnnotation`, which I think might be a little better than `Annotation(shared=True)` because it keeps the factories API more separate from the `models.Annotation` class (if we changed `models.Annotation` to no longer use a `shared` property, we wouldn't have to change every test that makes an annotation using a factory). But if we did that we should really rename `factories.Annotation` to `factories.PrivateAnnotation` too. Anyway, I haven't done it yet